lib/benchee/formatters/csv/statistics.ex

```diff
@@ -1,85 +1,106 @@
 defmodule Benchee.Formatters.CSV.Statistics do
+  @moduledoc """
+  Functionality for converting Benchee scenarios to csv with statistics.
+  """
+
   alias Benchee.{Statistics, Benchmark.Scenario}
 
-  @column_descriptors [
+  @headers_with_memory [
     "Name",
     "Input",
     "Iterations per Second",
-    "Average",
-    "Standard Deviation",
     "Standard Deviation Iterations Per Second",
-    "Standard Deviation Ratio",
-    "Median",
-    "Minimum",
-    "Maximum",
-    "Sample Size"
+    "Run Time Average",
+    "Run Time Median",
+    "Run Time Minimum",
+    "Run Time Maximum",
+    "Run Time Standard Deviation",
+    "Run Time Standard Deviation Ratio",
+    "Run Time Sample Size",
+    "Memory Usage Average",
+    "Memory Usage Median",
+    "Memory Usage Minimum",
+    "Memory Usage Maximum",
+    "Memory Usage Standard Deviation",
+    "Memory Usage Standard Deviation Ratio",
+    "Memory Usage Sample Size"
   ]
-  @moduledoc """
-    Functionality for converting Benchee scenarios to csv with statistics.
-  """
 
-  @doc """
-    Adds headers to given measurements.
+  @headers_without_memory [
+    "Name",
+    "Input",
+    "Iterations per Second",
+    "Standard Deviation Iterations Per Second",
+    "Run Time Average",
+    "Run Time Median",
+    "Run Time Minimum",
+    "Run Time Maximum",
+    "Run Time Standard Deviation",
+    "Run Time Standard Deviation Ratio",
+    "Run Time Sample Size"
```

Comment:
I think we could build `@headers_with_memory` from the `without_memory` stuff by just appending the memory header values  to avoid duplication